### DISCUSSION TITLE
Is vanilla extract included in the bundle file?

### DISCUSSION THREAD

**Discussion by dante01yoon** [vanilla-extract-css/vanilla-extract#911] [General]
Hi, thank you for this wonderful libraries. I have few questions below. 

1. Is vanilla extract included in the bundle file? (js bundle file)
  - I guess if there are api codes in component like sprinkles, should be included in js bundle files  for create classes during run time , 
but if only all apis are used in `.css.ts` files, no codes related with VE included in js bundle files.

2. If not included, how do individual files fall apart? (I wonder how this part should be handled during build and deployment)
  - guess depends on bundler(webpack, esbuild) config, files could be split into multiple css files or not

**Comment by DanielHeath** [comment]
There's no runtime JS at all, even in sprinkles. It's all build time. The CSS class names are embedded as strings in the built javascript files.

**Reply by dante01yoon** [reply]
I see. thanks for response. !
